src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -102,7 +102,7 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
             titleTranslationKey: 'workspace.moreFeatures.taxes.title',
             subtitleTranslationKey: 'workspace.moreFeatures.taxes.subtitle',
             isActive: (policy?.tax?.trackingEnabled ?? false) || isSyncTaxEnabled,
-            disabled: isSyncTaxEnabled || policy?.connections?.quickbooksOnline.data.country === CONST.COUNTRY.US,
+            disabled: isSyncTaxEnabled || policy?.connections?.quickbooksOnline?.data?.country === CONST.COUNTRY.US,
```

Comment:
This was crashing because `data`  was undefined.

Reply:
ah yep because we're not yet fetching the `connections` data when the More Features page is opened 👍

Reply:
I think this is being fixed the right way somewhere else, right? do you have a link to that?

Reply:
I think this is the same crash seen here: https://github.com/Expensify/App/issues/41129